src/index.html

```diff
@@ -4,14 +4,171 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <!--Personal info-->
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="Surname:"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field field" >
+          Name:
+          <input
+          type="text"
+          name="Name:"
+          autocomplete="off"
+          >
+        </label>
```

Comment:
1) apply code styling recommendations all over the page
2) write input `name` lowercased
3) add only common class to all your fields like `field`
4) apply those recommendations all over the page
```suggestion
        <label class="field" >
          Name:
          <input
            type="text"
            name="name"
            autocomplete="off"
          >
        </label>
```